lib/krikri/harvesters/api_harvester.rb

```diff
@@ -23,7 +23,7 @@ class ApiHarvester
     # @see .expected_opts
     def initialize(opts = {})
       super
-      @opts = opts.fetch(:api, {})
+      @opts = opts.fetch(:api, {}).symbolize_keys
```

Comment:
This changes the interface across the other `Krikri::Harvester` implementation classes. By default, I believe the implementation classes expect the outermost "layer" to have symbol-based keys.